crates/rpc/rpc-eth-api/src/core.rs

```diff
@@ -334,7 +334,8 @@ pub trait EthApi {
 #[async_trait::async_trait]
 impl<T> EthApiServer for T
 where
-    Self: FullEthApi,
+    T: FullEthApi,
+    jsonrpsee_types::error::ErrorObject<'static>: From<T::Error>,
```

Comment:
is this required? doesn't FullEthApi already enforce this?

Reply:
compiler isn't powerful enough to infer that